Screens/LandingPage.js

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import { View, Text, StyleSheet, Button } from 'react-native';
+
+
+const LandingPage = ({navigation}) => {
+
+    return (
+        <View style= {styles.container}>
+        
+            <Text style={styles.title}>
```

Comment:
Let's make this text bigger, since it feels like there's a bit too much whitespace

Reply:
the styles object down below also doesn't seem to have a "title" listing